integration-libs/punchout/root/guards/punchout-navigation.guard.ts

```diff
@@ -149,6 +147,9 @@ export class PunchoutNavigationGuard {
       take(1),
       switchMap((punchoutState: PunchoutState | undefined) => {
         if (punchoutState?.punchoutSessionId) {
+          if (cxRoute === 'logout') {
```

Comment:
put it in var same way as HOME_PAGE_URL, eg:
`protected readonly LOGOUT_CX_ROUTE='logout';`